Mage.Sets/src/mage/cards/m/MetamorphicAlteration.java

```diff
@@ -139,8 +139,8 @@ public boolean apply(Game game, Ability source) {
         for (Ability ability : copied.getAbilities()) {
             permanent.addAbility(ability, source.getSourceId(), game);
         }
-        permanent.getPower().setValue(copied.getPower().getBaseValueModified());
-        permanent.getToughness().setValue(copied.getToughness().getBaseValueModified());
+        permanent.getPower().setModifiedBaseValue(copied.getPower().getBaseValue());
+        permanent.getToughness().setModifiedBaseValue(copied.getToughness().getBaseValue());
```

Comment:
I'm pretty sure this should be baseValue not modifiedBaseValue since it's a copy effect.